packages/ui/src/working-groups/model/getRoleWarning.ts

```diff
@@ -0,0 +1,16 @@
+import { WorkerStatusTypename } from '../types'
+
+export function getRoleWarning(status: WorkerStatusTypename) {
+  if (status === 'WorkerStatusActive') return undefined
+  if (status === 'WorkerStatusLeaving')
+    return {
+      title: 'Unstaking period',
+      content: 'This role is in the unstaking period.',
+      isClosable: false,
+    }
+  return {
```

Comment:
Maybe let's keep `{}` for ifs:

```suggestion
  if (status === 'WorkerStatusActive') {
    return undefined
  }
  
  if (status === 'WorkerStatusLeaving') {
    return {
      title: 'Unstaking period',
      content: 'This role is in the unstaking period.',
      isClosable: false,
    }
  }
  
  return {
```